pkg/operator/vclib/connection.go

```diff
@@ -37,13 +111,13 @@ var (
 	clientLock sync.Mutex
 )
 
```

Comment:
Shouldn't these 2 declarations be on top of the file?

Reply:
yes.  I'll move these up.